### DISCUSSION TITLE
Coesistenza Homebridge ed Home Assistant

### DISCUSSION THREAD

**Discussion by Cupra1979** [indomus/forum#462] [HUB: Homebridge & Apple HomeKit]
Ciao a tutti!!

Stavo cercando di capire se e come possono coesistere Homebridge ed Home Assistant su un unico Raspberry.
Attualmente ho un Raspberry Pi 4 che uso prevalentemente per esporre parecchi device verso Homey (il mio HUB domotico principale). Ho visto che alcuni (ad es. Broadlink) sono gestibili più facilmente attraverso Homebridge e quindi mi domandavo se fosse possibile utilizzarlo sul mio Pi 4 già in uso (con pro e contro), oppure se fosse il caso di dedicargli un Raspberry.

L'alternativa è installarlo sul Mac Mini, ma poi ovviamente dovrebbe restare sempre in funzione...
Grazie mille per le eventuali risposte, ciao!

**Comment by snakuzzo** [comment]
Possono coesistere tranquillamente sullo stesso server...nessuna controindicazione

**Reply by Cupra1979** [reply]
Anche se Home Assistant è installato come Home Assistant OS? Ho scelto la strada più pratica ed immediata

**Reply by snakuzzo** [reply]
ovviamente no...in quel caso non puoi farli coesistere

**Reply by Scialla** [reply]
guarda che li gestisci meglio con HA che con HB...

**Reply by bubez81** [reply]
In realtà c'è modo di farli coesistere comunque basta installare homebridge tramite portrainer come docker in homeassistant

**Comment by KC-inDomus** [comment]
sbagli grandemente, credendo che Homebridge gestisca meglio broadlink di quanto non faccia HA.
è l'esatto contrario.
ti consiglio vivamente di approfondire meglio smartir e quanto puoi fare con i remote di HA che, eventualmente, poi esponi verso homekit. saltando a pie' pari Homebridge.

**Comment by Cupra1979** [comment]
Grazie a tutti per le risposte.
Purtroppo sono a livello abbastanza basic con HA e mi ha un po' frenato il "lavoro" che c'è dietro l'integrazione dei Broadlink. Per questo pensavo che il passaggio tramite Homebridge semplificasse l'integrazione. A dire il vero l'utilizzo di Homekit non è una necessità, ma invece uno "stratagemma" per esporre il tutto verso Homey, visto che l'app esistente per quell'HUB non è il massimo...
Provo ad approfondire il tema documentandomi su quanto consigliato, grazie.

**Comment by marcgrc** [comment]
Ciao a tutti
Io ho già un homebridge funzionante con svariati dispositivi
Posso far vedere io mio homebridge ad Home Assistant in modo che Home Assistant erediti i dispositivi dal mio homebridge ?
Grazie

**Reply by Scialla** [reply]
Risposta breve: No
Devi partire da 0; da HB puoi estrapolarti alcune parti della configurazione che ti saranno utili per integrare i tuoi dispositivi in HA, ma non esiste nulla per importare tutto automaticamente

**Reply by marcgrc** [reply]
Grazie
